proposals/incubation/kubeflow.md

```diff
@@ -0,0 +1,209 @@
+    * [Hougangliu](https://github.com/Hougangliu) - Hougang Liu (IBM)
+    * [Johnugeorge](https://github.com/Johnugeorge) - Johnu George (Nutanix)
+
+## Infrastructure requests (CI/CNCF Cluster)
+
+Kubeflow uses AWS for its CI needs. The community is already working on streamlining the infrastructure provisioning and operations ([https://github.com/kubeflow/testing/issues/1006](https://github.com/kubeflow/testing/issues/1006)). If Kubeflow does get accepted for incubation in CNCF, the community will work towards migrating the ownership of this infrastructure to the CNCF.
+
+Each distribution offering Kubeflow as a service operates their own test/build/release infrastructure. These will continue being the responsibility of each provider and will not transition to CNCF ownership. The maintainers of these infrastructures are available in the platform-specific Slack channels:
+
+* AWS - [https://kubeflow.slack.com/archives/CKBA5D0MU](https://kubeflow.slack.com/archives/CKBA5D0MU)
+* Azure - [https://kubeflow.slack.com/archives/CUW6SLCPR](https://kubeflow.slack.com/archives/CUW6SLCPR)
+* Platform - [https://kubeflow.slack.com/archives/CKH7V1M7F](https://kubeflow.slack.com/archives/CKH7V1M7F)
+
+## Communication Channels
+
+The Kubeflow community includes a variety of Slack forums for general discussion, feature requests, job postings, and sub-project, platform, or geo-specific channels. Popular Kubeflow Slack channels are listed here - [https://www.kubeflow.org/docs/about/community/#kubeflow-slack](https://www.kubeflow.org/docs/about/community/#kubeflow-slack)
+
+
+## Issue tracker
+
+* Kubeflow - [https://github.com/kubeflow/kubeflow/issues](https://github.com/kubeflow/kubeflow/issues)
+* Pipelines - [https://github.com/kubeflow/pipelines/issues](https://github.com/kubeflow/pipelines/issues)
+* Training-operator -  [https://github.com/kubeflow/training-operator/issues](https://github.com/kubeflow/training-operator/issues)
+* Katib - [https://github.com/kubeflow/katib/issues](https://github.com/kubeflow/katib/issues)
+
+## Community
+
+* Repo - [https://github.com/kubeflow/community](https://github.com/kubeflow/community)
+* Website - [https://www.kubeflow.org/docs/about/community/](https://www.kubeflow.org/docs/about/community/)
+* Workgroups & SIGs - [https://github.com/kubeflow/community/blob/master/wg-list.md](https://github.com/kubeflow/community/blob/master/wg-list.md)
+
+## Code of Conduct
+
+[https://github.com/kubeflow/community/blob/master/CODE_OF_CONDUCT.md](https://github.com/kubeflow/community/blob/master/CODE_OF_CONDUCT.md)
+
+## Social media accounts
+
+* Youtube - https://www.youtube.com/kubeflow
+* Twitter - https://twitter.com/kubeflow
+* LinkedIn - [https://www.linkedin.com/company/kubeflow](https://www.linkedin.com/company/kubeflow)
+* Google Groups - [https://groups.google.com/g/kubeflow-discuss](https://groups.google.com/g/kubeflow-discuss)
+* Google Workspace (used to manage community calendar, Drive, etc.) 
+
```

Comment:
* Kubeflow blog - https://blog.kubeflow.org/

Reply:
Good catch. I've added it under community assets though, it's a KF property, not like the other social media ones listed here.